cmd/crates/soroban-test/tests/it/integration/cookbook.rs

```diff
@@ -109,27 +110,11 @@ fn run_command(
     }
 
     println!("Executing command: {} {}", cmd, modified_args.join(" "));
-    let result = sandbox.new_assert_cmd(&cmd).args(&modified_args).assert();
-
-    if command.contains("keys generate") {
-        result
-            .code(predicates::ord::eq(0).or(predicates::ord::eq(1)))
-            .stderr(
-                predicate::str::is_empty().or(predicates::str::contains("Generated new key for")
-                    .or(predicates::str::contains("The identity")
-                        .and(predicates::str::contains("already exists")))),
-            );
-    } else if command.contains("contract invoke") {
-        result
-            .failure()
-            .stderr(predicates::str::contains("error: unrecognized subcommand"));
-    } else if command.contains("contract restore") {
-        result
-            .failure()
-            .stderr(predicates::str::contains("TxSorobanInvalid"));
-    } else {
-        result.success();
-    }
+    sandbox
+        .new_assert_cmd(&cmd)
+        .args(&modified_args)
+        .assert()
+        .success();
```

Comment:
This change make it so that every command executed is expected to succeed. For commands that we don't expect to succeed, they are instead marked as 'ignore' in the markdown itself. The author can decide what gets tested and what doesn't for those edge cases that are just impractical.